x-pack/plugins/apm/server/lib/transactions/get_transaction/index.ts

```diff
@@ -60,3 +66,26 @@ export async function getTransaction(
   const resp = await client<Transaction>('search', params);
   return oc(resp).hits.hits[0]._source();
 }
+
+export async function getTransactionWithErrorCount(
+  transactionId: string,
+  traceId: string | undefined,
+  setup: Setup
+): Promise<TransactionWithErrorCount> {
+  const transaction = await getTransaction(transactionId, traceId, setup);
+
+  if (transaction) {
+    return {
+      transaction,
+      errorCount: await getErrorCountByTransaction({
+        setup,
+        transaction
+      })
+    };
+  }
+
+  return {
+    transaction: undefined,
+    errorCount: 0
+  };
+}
```

Comment:
If we want this as a separate method, this should also be a separate file.
But I'm thinking this can just go in the `getTransaction` method, since it'll only be used in a single place anyway (after I've removed it from the errors endpoint).